doc/lightning-addgossip.7.md

```diff
@@ -1,31 +1,54 @@
 lightning-addgossip -- Command for injecting a gossip message (low-level)
-===============================================================
+=========================================================================
 
 SYNOPSIS
```

Comment:
This commit needs a Changelog message!

Changelog-Added: doc: Major command reference rework, including examples and documentation generated from the fully-tested JSON schemas.

Reply:
Nope, I see you did this later, and even better :orange_heart:

Reply:
Updated the changelog on this page too.